pydarn/plotting/rtp.py

```diff
@@ -399,7 +399,7 @@ def plot_range_time(cls, dmap_data: List[dict], parameter: str = 'v',
         ax.set_ylim(ymin, ymax)
 
         if slant:
-            ax.yaxis.set_ticks(np.arange(ymin, ymax+1, yspacing))
+            ax.yaxis.set_ticks(np.arange(np.ceil(ymin / 100.0) * 100, ymax+1, yspacing))
```

Comment:
```suggestion
            ax.yaxis.set_ticks(np.arange(np.ceil(ymin/100.0)*100, 
                                                            ymax+1, yspacing))
```
Not entirely sure how indented that needs to be